tests/modmail/conftest.py

```diff
@@ -6,6 +6,11 @@
 import pytest
 
 
+_ORIG_ENVIRON = None
+
+_modmail_env_prefix = "MODMAIL_"
```

Comment:
This use `modmail.config.ENV_PREFIX` once config PR is merged, worth adding a `TODO`